cmd/influx_tsm/stats/stats.go

```diff
@@ -18,30 +18,37 @@ type Stats struct {
 	TotalTime       time.Duration
 }
 
+// AddPointsRead adds read points
```

Comment:
Can you make this docstring more useful with something like:

```
// AddPointsRead increments the number of read points.
```

Same with the other docstrings below this. They should also have periods at the end of each sentence.